index.js

```diff
@@ -0,0 +1,93 @@
+$(document).ready(()=>{
+  let loadTrips = function loadTrips() {
+    $('#load').hide('slow')
+
+    $.get('https://trektravel.herokuapp.com/trips',
+    (response) => {
+      response.forEach(function(trip) {
+        let tripInfo = `<div class = "inside" id=${trip.id}> <h3 id=${trip.id} > ${trip.name} </h3> </div>`;
+
+        $('#single-trip').html('');
+        $(tripInfo).appendTo('#many-trips').show('slow');
+
+      });
+    })
+    .fail(function(response){
+      console.log(response);
+      $('#fail').html('<p>Request was unsuccessful</p>')
+    });
+  };
+
+  let loadTrip = function loadTrip(id){
+    $.get(`https://trektravel.herokuapp.com/trips/${id}`,
+      (response) => {
+        console.log(response);
+        let tripInfo =
+        `<p> ID: ${response.id} </p>
+        <p> Name: ${response.name} </p>
```

Comment:
These trip details should probably be wrapped in a sectioning element like `<article>`. Even dynamically generated HTML needs to be well-structured and semantic.